core/server/models/post.js

```diff
@@ -878,6 +878,9 @@ Post = ghostBookshelf.Model.extend({
         // CASE: never expose the revisions
         delete attrs.mobiledoc_revisions;
 
+        // CASE: hide the newsletter_id for now
+        delete attrs.newsletter_id;
```

Comment:
Arguably this could be done in the serialiser - but I think here works too 👍

Reply:
I wanted to avoid any side-effect in code in general for this PR, I'll look into the serializer next, thanks!